yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -65,7 +54,7 @@ func init() {
 	// defining flags
 	assessMigrationBulkCmd.Flags().StringVar(&fleetConfigPath, "fleet-config-file", "", "File containing the connection params for schema(s) to be assessed (required)")
 	BoolVar(assessMigrationBulkCmd.Flags(), &continueOnError, "continue-on-error", true, "If true, it will print the error message on console and continue to next schema’s assessment")
-	assessMigrationBulkCmd.Flags().Bool("ignore-exists", true, "If true, skip assessment if for a schema it's already done")
+	BoolVar(assessMigrationBulkCmd.Flags(), &ignoreExists, "ignore-exists", true, "If true, skip assessment if for a schema it's already done")
```

Comment:
Let's discuss this once :D I still don't see why we need ignoreExists.

Reply:
yeah i am also in favour of removing this now, don't see any important use case for it as such.

Good to have our default behaviour to ignore the existing ones, that will also keep things simple for users.

Reply:
done